// The message of flows.

syntax = "proto3";
package ns3_proto;

message Flow {
	uint32 srcNode = 1;
	uint32 srcPort = 2;
	uint32 dstNode = 3;
	uint32 dstPort = 4;
	uint32 size = 5;
	uint32 arriveTime = 6; // in microseconds
	optional uint32 priority = 7; 
}

message Flows {
	repeated Flow flows = 1;
}